src/app/cards/munchkin/cards/treasures/disposable-item/model/disposable-item-type.ts

```diff
@@ -0,0 +1,13 @@
+import { CardType, LayoutType } from 'src/app/cards/infrastructure/cards';
+
+export const DisposableItemType: CardType = {
+  name: 'disposableItem',
+  namePlural: 'disposableItems',
+  displayName: 'Одноразова манатка',
```

Comment:
This gives the impression of a _single use_ item rather than _disposable_. What do you think about changing the name? https://www.oxfordlearnersdictionaries.com/definition/english/single-use
https://diffsense.com/diff/disposable/single-use#:~:text=The%20difference%20between%20Disposable%20and,then%20discarded%20or%20disposed%20of.&text=Disposable%20as%20a%20noun%3A,rather%20than%20refilled%20or%20repaired.

Reply:
Makes sense
Any suggestions to change name for permanent item to be consistent?

Reply:
What about `reusable item`?